includes/REST/OrderController.php

```diff
@@ -493,6 +493,7 @@ protected function get_order_item_data( $item ) {
         // Add SKU and PRICE to products.
         if ( is_callable( array( $item, 'get_product' ) ) ) {
             $data['sku']   = $item->get_product() ? $item->get_product()->get_sku() : null;
+            $data['image'] = wp_get_attachment_image_url( $item->get_product()->get_image_id() ) ? wp_get_attachment_image_url( $item->get_product()->get_image_id(), 'full' ) : wc_placeholder_img_src();
```

Comment:
if product is deleted, we can't access `$item->get_product()->get_image_id()` hence will trigger fatal error, kindly add proper checking here.